### PR TITLE
Update qgis_configuration.rst (general settings tab update)

### LINKED ISSUE #2421: [FEATURE] Allow overwriting the project home path
Original commit: https://github.com/qgis/QGIS/commit/4e5c08e2b54b7f3b06d15f4538aba6a8d9b84ae7 by nyalldawson

This allows the project home path (which is used by the browser
to create the 'Project Home' item) to be set by users for a
project, instead of always matching the location where the project
is saved.

This allows users to set the project home to a folder which contains
data and other content, and is especially useful for organisations
where qgis projects are not stored in the root folder of a organisational
'project'.

Project home paths can also be set to relative paths, in which
case they will be relative to the project saved location.

The path can be set through the Project Properties dialog, or
by right-clicking on the Project Home browser item and
selecting 'set project home'

Sponsored by SMEC/SJ

### PR DESCRIPTION
**Pull request by havatv**
Update the general settings of the general tab of project properties. Fixes #2421.

### CODE REVIEW

**Comment by DelazJ** [comment]
Don't you want to be more verbose on what the project home feature is? Quoting Nyall : "a folder which contains data and other content" necessary for the project.
Also, we should mention that 
> Project home paths can also be set to relative paths, in which case they will be relative to the project saved location.

I wonder if it's done in setting the "save path**S**" drop-down (in which case line 885 should be updated (not only the layers are relative)) or if you should add the path in a relative form.

**Comment by havatv** [comment]
I was trying to keep it short. Probably a good idea to be more verbose - I will try.
Regarding the Save paths - I guess this has to do with how paths to data sources are saved in the project file.

File: source/docs/user_manual/introduction/qgis_configuration.rst [hunk 1]
```diff
@@ -876,6 +876,9 @@ General Properties
 
 In the :guilabel:`General` tab, the :guilabel:`General settings` let you:
 
+* see the location of the project file
+* set the folder for the project home (available in the `Project Home` item
+  in the browser)
```
Comment (havatv) [inline source/docs/user_manual/introduction/qgis_configuration.rst]:
```suggestion
  in the browser).
  The path can be relative to the folder of the project file (type it in) or absolute
  (type or choose).
  The project home can be used for storing data and other content that is
  necessary or useful for the project.
```

Comment (havatv) [reply source/docs/user_manual/introduction/qgis_configuration.rst]: Better, @DelazJ ?

File: source/docs/user_manual/introduction/qgis_configuration.rst [hunk 2]
```diff
@@ -876,6 +876,13 @@ General Properties
 
 In the :guilabel:`General` tab, the :guilabel:`General settings` let you:
 
+* see the location of the project file
+* set the folder for the project home (available in the `Project Home` item
```
Comment (DelazJ) [inline source/docs/user_manual/introduction/qgis_configuration.rst]: This needs to be doubled
```suggestion
* set the folder for the project home (available in the ``Project Home`` item
```

File: source/docs/user_manual/introduction/qgis_configuration.rst [hunk 3]
```diff
@@ -876,6 +876,13 @@ General Properties
...
+  in the browser).
+  The path can be relative to the folder of the project file (type it in) or absolute
+  (type or choose).
```
Comment (DelazJ) [inline source/docs/user_manual/introduction/qgis_configuration.rst]:
```suggestion

```

File: source/docs/user_manual/introduction/qgis_configuration.rst [hunk 4]
```diff
@@ -876,6 +876,13 @@ General Properties
 
 In the :guilabel:`General` tab, the :guilabel:`General settings` let you:
 
+* see the location of the project file
+* set the folder for the project home (available in the `Project Home` item
+  in the browser).
+  The path can be relative to the folder of the project file (type it in) or absolute
```
Comment (DelazJ) [inline source/docs/user_manual/introduction/qgis_configuration.rst]: Maybe, we could make this a bit compact. I'm not sure of the need of the precision in parenthesis
```suggestion
  The path can be entered as relative to the project's folder or as an absolute path.
```

File: source/docs/user_manual/introduction/qgis_configuration.rst [hunk 5]
```diff
@@ -876,6 +876,13 @@ General Properties
...
+  (type or choose).
+  The project home can be used for storing data and other content that is
+  necessary or useful for the project.
```
Comment (DelazJ) [inline source/docs/user_manual/introduction/qgis_configuration.rst]:
```suggestion
  useful for the project.
```
?

File: source/docs/user_manual/introduction/qgis_configuration.rst [hunk 6]
```diff
@@ -876,6 +876,12 @@ General Properties
 
 In the :guilabel:`General` tab, the :guilabel:`General settings` let you:
 
+* see the location of the project file
+* set the folder for the project home (available in the ``Project Home`` item
+  in the browser).
+  The path can be relative to the folder of the project file (type it in) or absolute
```
Comment (havatv) [inline source/docs/user_manual/introduction/qgis_configuration.rst]:
```suggestion
  The path can be relative to the folder of the project file (type it in) or absolute.
```

File: source/docs/user_manual/introduction/qgis_configuration.rst [hunk 7]
```diff
@@ -876,6 +876,12 @@ General Properties
 
 In the :guilabel:`General` tab, the :guilabel:`General settings` let you:
 
+* see the location of the project file
+* set the folder for the project home (available in the ``Project Home`` item
+  in the browser).
+  The path can be relative to the folder of the project file (type it in) or absolute.
```
[the remark this answers is not in the document]
Comment (DelazJ) [reply source/docs/user_manual/introduction/qgis_configuration.rst]: > I think we should mention how relative paths are specified.

reason why I was suggesting "entered"
If no better word, let's merge by the end of the day (in utc of course!)

[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/introduction/qgis_configuration.rst]: OK.  I suggest that you merge.

**Pull request merged**